packages/tailwind/docs/Shadow.stories.tsx

```diff
@@ -1,13 +1,12 @@
 import React from "react"
 import { Story } from "@storybook/react"
 import { Divider } from "@kaizen/draft-divider"
+import { kaizenTailwindTheme } from "@kaizen/tailwind"
 import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 import styles from "./styles.module.scss"
 
-import { kaizenTailwindTheme } from "@kaizen/tailwind"
-
 export default {
-  title: "Tailwind/Effects/Shadow",
+  title: "Tailwind/Effects/Box Shadow",
```

Comment:
Just changing this to box shadow to make it easier to find it a dev is searching for its equivalent